src/warplib/implementations/external_input_checks/externalInputChecksInts.ts

```diff
@@ -5,31 +5,19 @@ const INDENT = ' '.repeat(4);
 export function external_input_check_ints(): WarplibFunctionInfo {
   return {
     fileName: 'external_input_check_ints',
-    imports: [
-      'from starkware.cairo.common.math_cmp import is_le_felt',
-      'from starkware.cairo.common.uint256 import Uint256',
-    ],
+    imports: [],
     functions: forAllWidths((int_width) => {
       if (int_width === 256) {
         return [
-          `func warp_external_input_check_int256{range_check_ptr}(x : Uint256){`,
-          `${INDENT}alloc_locals;`,
-          `${INDENT}let inRangeHigh : felt = is_le_felt(x.high, ${mask(128)});`,
-          `${INDENT}let inRangeLow : felt = is_le_felt(x.low, ${mask(128)});`,
-          `${INDENT}with_attr error_message("Error: value out-of-bounds. Values passed to high and low members of Uint256 must be less than 2**128."){`,
-          `${INDENT.repeat(2)}assert 1 = (inRangeHigh * inRangeLow);`,
-          `${INDENT}}`,
-          `${INDENT}return();`,
+          `fn warp_external_input_check_int256(x : u256) {`,
+          `${INDENT}return ();`,
           `}\n`,
         ].join('\n');
       } else {
         return [
-          `func warp_external_input_check_int${int_width}{range_check_ptr}(x : felt){`,
-          `${INDENT}let inRange : felt = is_le_felt(x, ${mask(int_width)});`,
-          `${INDENT}with_attr error_message("Error: value out-of-bounds. Value must be less than 2**${int_width}."){`,
-          `${INDENT.repeat(2)}assert 1 = inRange;`,
-          `${INDENT}}`,
-          `${INDENT}return ();`,
+          `fn warp_external_input_check_int${int_width}(x : felt){`,
+          `${INDENT}let max: felt = ${mask(int_width)};`,
+          `${INDENT}assert( x <= max, 'Error: value out-of-bounds.');`,
```

Comment:
I am no longer sure we need to do external input check for anything. I think their added automatically by the cairo1

Reply:
Same as first comment response, compatibility issues.

Reply:
Ok